src/main/scad/laszlo-molnar/l-i-vator-i/molnar.l-i-vator-i.scad

```diff
@@ -35,11 +35,10 @@ module box() {
 
 module pieces() {
     burr_plate([
-        ["xx{connect=mz+x+,clabel=C}"],
-        ["x.|x{connect=fz+y-,clabel=B}."],
-        ["x{connect=mz+y-,clabel=B}xx{connect=fy+z+,clabel=C}"],
         ["x.|x{connect=fz+y-,clabel=A}."],
         ["x{connect=mz+y-,clabel=A}xx|..x"],
+        ["xx{connect=mz+x-,clabel=B}"],
+        ["xx|x.","..|x.","..|x{connect=fx-y+,clabel=B}."],
```

Comment:
connectors on the x- face need to be pointing in the z+ direction or they won't print accurately (due to the flat overhang).

Changing this to f=x-z+ and rotating the corresponding male connector accordingly would fix this.

Reply:
Good suggestion, updated
![image](https://user-images.githubusercontent.com/1962079/75350917-4283ab80-58e2-11ea-9776-a01455c3a5df.png)